applications/nrf_desktop/src/modules/usb_state.c

```diff
@@ -1021,6 +1113,14 @@ static void usb_init_next_status_cb(struct usbd_context *const usbd,
 		break;
 	}
 
+	if (IS_ENABLED(CONFIG_DESKTOP_USB_STACK_NEXT_DISABLE_ON_VBUS_REMOVAL)) {
+		err = handle_usbd_state_on_status_change(msg->type);
```

Comment:
Please keep `err` locally in this scope

Reply:
You actually may not emit an extra log here, `handle_usbd_state_on_status_change` already does it

Reply:
I'll keep the error log.